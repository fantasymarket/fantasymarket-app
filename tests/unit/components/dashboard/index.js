import React from 'react';
import Dashboard from '@components/dashboard';
import { render } from '@testing-library/react';

describe('Dashboard component', () => {
	const container = render(<Dashboard />);

	it('renders the component', () => {
		expect(container.baseElement).toMatchSnapshot();
	});
});
