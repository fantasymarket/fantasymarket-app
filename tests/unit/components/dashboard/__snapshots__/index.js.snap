// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard component renders the component 1`] = `undefined`;
